packages/dapp/components/utils/local-data.ts

```diff
@@ -0,0 +1,40 @@
+import dotenv from "dotenv";
+import { getUbiquityManagerContract } from "@/components/utils/contracts";
+import { ethers } from "ethers";
+import { LOCAL_NODE_ADDRESS } from "@/components/lib/hooks/use-web-3";
+
+dotenv.config();
+
+export async function fetchData() {
+  const provider = new ethers.providers.JsonRpcProvider(LOCAL_NODE_ADDRESS);
+  const signer = provider.getSigner();
+
+  if (process.env.DEBUG === "true") {
+    console.log(signer);
+  }
+
+  if (provider && signer) {
+    try {
+      const managerAddress = "0x4e037B9A8Ce977462DA4E10Fc164363C827abfc6";
+      const managerContract = await getUbiquityManagerContract(managerAddress, provider);
+
+      if (process.env.DEBUG === "true") {
+        console.log(await managerContract.connect(signer).twapOracleAddress(), "Twap Oracle Address");
+        console.log(await managerContract.connect(signer).creditCalculatorAddress(), "Credit Calculator Address");
+      }
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  //if (process.env.DEBUG === 'true') {
+  //console.log(provider);
+  const block = await provider?.getBlockNumber();
```

Comment:
it might be set to debug by default but leaving open here for the client-side to show anvil chainId

Reply:
i'm in favor of showing up the chainId on the client side, i think if you're going multi-chain it's cool and good